python: can't open file 'parallel_ins.py': [Errno 2] No such file or directory
